src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -669,6 +678,33 @@ internal sealed override OverriddenOrHiddenMembersResult OverriddenOrHiddenMembe
             }
         }
 
+        public override string MetadataName
+        {
+            get
+            {
+                return (object)_replacedBy == null ?
+                    base.MetadataName :
+                    GeneratedNames.ReplacedMemberName(Name, uniqueId: 0);
```

Comment:
The `Replaced` and `ReplacedBy` members reference each other. The `ReplacedBy` member is included in `GetMembers()` of the containing type while the `Replaced` member is only available from `ReplacedBy.Replaced`.

Added comments to `Symbol.Replaced` and `Symbol.ReplacedBy`.